src/app/_data/homepage/filecoinEcosystemData.ts

```diff
@@ -1,15 +1,33 @@
 import { Code, HardDrives, Money, Person } from '@phosphor-icons/react/dist/ssr'
 
+import type { IconProps } from '@/components/Icon'
+
 import { PATHS } from '@/constants/paths'
+import { FILECOIN_DOCS_URLS } from '@/constants/siteMetadata'
+
+interface FilecoinEcosystemData {
+  heading: {
+    title: string
+    icon: IconProps['component']
+  }
+  description: string
+  cta: {
+    href: string
+    text: string
+  }
+}
```

Comment:
in what cases do we prefer interface over type? Sorry if I missed a discussion on this

Reply:
Prefer `interface` when you need to define an object type that might be extended or merged, as interfaces support declaration merging and are more flexible for hierarchical design. Use `type` for defining unions, intersections, or more complex types, as it’s more versatile for combining types and creating aliases.